Add clubs page

kewbish opened the pull request:
This PR adds a clubs page (#94):
- The layout of the page is copied from the Services page, with each club's logo and a tagline from their website as a description
    - Let me know if there are other clubs that should be included! 
    - (The ACM logo is not actually their logo, I couldn't find one on their site, Facebook, or Discord anywhere)
- As Asad mentioned, I included a couple engineering Design Teams, but because there're so many, I just linked the ones we mentioned in our last meeting and included a link to the main Design Teams listing  
- The page is linked both in the `/about` page, and at the bottom footer under `Resources` - let me know if you'd like it moved elsewhere.

eqfy commented on content/about/clubs.md:
```diff
@@ -0,0 +1,66 @@
+---
+title: Affiliated Clubs
```
Instead of saying "affiliated", we should probably say "partner" clubs

kewbish replied:
Sounds good, will change!

eqfy commented on content/about/clubs.md:
```diff
@@ -0,0 +1,66 @@
+# Card images should all be 348x196
+---
+
+The UBC CSSS is affiliated with the following UBC CS-related clubs.
```
I guess same idea as above, we should probably say `The following UBC CS-related clubs are partners of the CSSS.`

kewbish replied:
Will do!

eqfy reviewed (approved):
Looks great! Thanks so much @kewbish!

In regards to your comments:
- ACM doesn't have a logo. In fact, I asked one of their execs about it a couple of months ago and he doesn't know either. So I used the exact logo that you added 😄 . 
- Since there are a lot of design teams, I personally think just leaving a link to the engineering design team page is probably better. What are your thoughts @AnimeAllstar ?
- Having the link in the /about page is fine. Though maybe it might be a good idea to have it on the top navigation bar instead?

kewbish commented:
Alright, I've:
- Reworded affiliated ⇒ partner clubs.
- Removed the design teams, and just kept the link to the engineering list.
- Removed the link within the `/about` page and moved it to the header.
@eqfy - let me know if there are any other changes to be made!

eqfy commented:
Looks good! Feel free to merge!
